redisinsight/ui/src/components/csrf/Csrf.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+
+import apiService, { setCSRFHeader } from 'uiSrc/services/apiService'
+import PagePlaceholder from '../page-placeholder'
+
+const csrfEndpoint = () => process.env.RI_CSRF_ENDPOINT || ''
```

Comment:
if it's a function it should start with "get". Or you can change this to a variable

Reply:
updated